src/Components/WebAssembly/Sdk/src/targets/Microsoft.NET.Sdk.BlazorWebAssembly.Current.targets

```
@@ -283,6 +283,11 @@ Copyright (c) .NET Foundation. All rights reserved.
       <_BlazorBuildBootJsonPath>$(IntermediateOutputPath)blazor.boot.json</_BlazorBuildBootJsonPath>
     </PropertyGroup>
 
+    <Error
+      Text="Unable to find $(OutDir)$(_BlazorOutputPath)%(BlazorWebAssemblyLazyLoad.Identity) to be lazy loaded later (are you missing an assembly reference?)"
```

```suggestion
      Text="Unable to find $(OutDir)$(_BlazorOutputPath)%(BlazorWebAssemblyLazyLoad.Identity) to be lazy loaded later. Confirm that project or package references are included and the reference is used in the project."
```

This seems like a more helpful error message.

---

Agree, will change it